tree/tree/src/TBranchElement.cxx

```diff
@@ -1143,10 +1143,12 @@ void TBranchElement::BuildTitle(const char* name)
 {
    TString branchname;
 
-   Int_t nbranches = fBranches.GetEntries();
+   Int_t nbranches = fBranches.GetEntriesFast();
 
    for (Int_t i = 0; i < nbranches; ++i) {
       TBranchElement* bre = (TBranchElement*) fBranches.At(i);
+      if (!bre)
```

Comment:
How can that happen, why is this not an error / inconsistency that should be diagnosed?

Reply:
bah ... "Technically" the user can screw with the list of branches ..